tests/utils/shopper.js

```diff
@@ -293,5 +300,50 @@ export const shopper = {
 				}
 			);
 		},
+
+		setCartQuantity: async ( productTitle, quantityValue ) => {
+			const cartItemXPath = getCartItemExpression( productTitle );
+			const quantityInputXPath =
+				cartItemXPath + '//' + getQtyInputExpression();
+
+			const [ quantityInput ] = await page.$x( quantityInputXPath );
+			await quantityInput.focus();
+			await pressKeyWithModifier( 'primary', 'a' );
```

Comment:
You can use `await expect( page ).toFill()` which will clear the input before typing in it.

Reply:
We can't use `toFill()` function in this case. We will need to use `CSS Selectors` to target the right input. And we can't do that when we have multiple elements in the cart, so we use `XPath` instead.

Reply:
Ok understood 👍